Backend/memo_places/tests.py

```diff
@@ -1,3 +1,148 @@
 from django.test import TestCase
+import requests
+import pytest
 
-# Create your tests here.
+class UserTests(TestCase):
+    id_created=None
+
+    def setUp(self):
+        data = {'email': 'test9998@test.pl',
+                'username':'XxTestxX',
+                'password': 'Test1234!'
```

Comment:
Nie powinien być jeszcze znak specjalny w haśle? Wydaje mi się że w kodzie jest że wymagany jest znak specjalny.

Reply:
Ok a „!” to nie jest znak specjalny.
Dodatkowo na backendzie i tak tego nie sprawdzamy